src/lib.rs

```diff
@@ -70,6 +72,10 @@ mod ffi {
         type MemoryQueueClient;
 
         fn new_memory_queue_client() -> Box<MemoryQueueClient>;
+        
+        fn new_memory_queue_client_with_random_events(
```

Comment:
This seems to be an API just for testing purpose, is is possible for us to move it elsewhere?